azure-iot-device/azure/iot/device/provisioning/pipeline/pipeline_adapter.py

```diff
@@ -0,0 +1,172 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import logging
+from azure.iot.device.common.pipeline import pipeline_stages_base
+from azure.iot.device.common.pipeline import pipeline_ops_base
+from azure.iot.device.common.pipeline import pipeline_stages_mqtt
+from azure.iot.device.provisioning.pipeline import (
+    pipeline_stages_provisioning,
+    pipeline_stages_provisioning_mqtt,
+)
+from azure.iot.device.provisioning.pipeline import pipeline_events_provisioning
+from azure.iot.device.provisioning.pipeline import pipeline_ops_provisioning
+
+logger = logging.getLogger(__name__)
+
+
+class PipelineAdapter(object):
```

Comment:
>PipelineAdapter [](start = 6, length = 15)

may be this file is not intended to be a part of PR. 
I am pretty sure I changed the name of this file and class to be provisioning pipeline.